### PR TITLE
Gj debug game

### PR DESCRIPTION
**Pull request by grace-joh**
## Describe your changes
- update #output_goodbye to print goodbye message before clearing screen
- add new lines between welcome, new board, and turn message (#output_welcome and #output_turn)
- capitalize outputs in Turn #valid?
- add sleep time to #output_winner
- rearrange system clear to print 'Play again?' with start screen
- add feature and tests to #reset Board
  - include the new function in Game to reset Board in #output_winner
-  reset the Turn @winner along with board reset
  - change instance variable winner to attr_accessor to reset

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added/updated corresponding tests.

@margdaly Thanks for playing and debugging it with me! So fun!!

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -84,4 +84,8 @@ def diagonal_arrays
     end
     diagonal_arrays
   end
-end
\ No newline at end of file
+
+  def reset
```
Comment (margdaly) [inline lib/board.rb:88]: Thank you for trying it with a new object first. This works much better

File: lib/game.rb [hunk 2]
```diff
@@ -34,13 +34,15 @@ def output_start
   end
 
   def output_goodbye
+    puts 'See ya later!'
```
Comment (margdaly) [inline lib/game.rb:37]: good switch- you couldn't see the bye after the exit too well and it leaves the terminal with a clean slate

File: lib/turn.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class Turn
-  attr_reader :board, :winner
+  attr_reader :board
+  attr_accessor :winner
```
Comment (margdaly) [inline lib/turn.rb:3]: It was accessor after all!

File: spec/board_spec.rb [hunk 4]
```diff
@@ -430,4 +430,20 @@
...
+      board.layout[:A] = ['X', 'X', 'X', 'O', '*', '*']
+      board.layout[:B] = ['X', 'X', 'O', '*', '*', '*']
+      board.layout[:C] = ['O', 'O', '*', '*', '*', '*']
+      board.layout[:D] = ['O', 'O', '*', '*', '*', '*']
+      board.layout[:E] = ['X', 'X', 'O', '*', '*', '*']
+      board.layout[:F] = ['O', 'O', 'X', '*', '*', '*']
+      board.layout[:G] = ['X', 'X', 'O', '*', '*', '*']
+      board.reset
+
+      expect(board.layout.values.flatten).to all(eq('*'))
```
Comment (margdaly) [inline spec/board_spec.rb:446]: This is my favorite test so far!

**Review by margdaly — APPROVED** [review]
Debugging was so fun to do with you! Haven't gotten tired of the game yet

File: lib/game.rb (hunk 2 above)
Comment (grace-joh) [reply lib/game.rb:37]: i agree! glad we changed it!

File: lib/board.rb (hunk 1 above)
Comment (grace-joh) [reply lib/board.rb:88]: haha yeah, it doesn't feel as convenient(?) but it works!

**Pull request merged**